### PR TITLE
Documentation naming fixes, spelling fixes, line length standardization.

### PR DESCRIPTION
**Pull request by derekbjohnson**
I was reading through the documentation and noticed a few misspelled words.  Fixed them as I noticed them.  Thanks for the great documentation.  Please let me know if I can help in any other way.

### CODE REVIEW

**Comment by ahsonkhan** [comment]
> I was reading through the documentation and noticed a few misspelled words. Fixed them as I noticed them. Thanks for the great documentation. Please let me know if I can help in any other way.

Thanks for making these fixes! Much appreciated.

File: docs/workitems.md [hunk 1]
```diff
@@ -1,109 +1,127 @@
-# Work Items
```
Comment (ahsonkhan) [inline docs/workitems.md]: Was this change intentional? It seems like you replaced the entire workitems doc with a copy of the roadmaps doc.

File: docs/specs/encoding.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
 # Encoding
 
-The existing .NET encoding APIs (System.Text.Encoding) don't work super well in non-allocating data pipelines:
+The existing .NET encoding APIs (System.Text.Encoding) don't work super well in
+non-allocating data pipelines:
```
Comment (ahsonkhan) [inline docs/specs/encoding.md]: What is the line length you are using here? What do we get from standardizing it (just curios)?

**Comment by KrzysztofCwalina** [comment]
Looks good. Thanks for the fixes @derekbjohnson!

File: docs/workitems.md (hunk 1 above)
Comment (derekbjohnson) [reply docs/workitems.md]: Sorry!  When I started reading I hadn't forked and then I made an error in copying the file changes.  I've fixed the file now.  Thank you!

File: docs/specs/encoding.md (hunk 2 above)
Comment (derekbjohnson) [reply docs/specs/encoding.md]: Sure!  The standard seems to be 80 characters in most repositories.  This helps keep the line length readable on older devices and devices with smaller screens.  Some editors (like Atom) will default the line length to 80.

**Comment by ahsonkhan** [comment]
Congrats on your first contribution. Looking forward to more :)
